src/Stratis.Bitcoin.Features.Consensus/ProvenBlockHeaders/ProvenBlockHeaderStore.cs

```diff
@@ -188,6 +188,8 @@ public async Task<ProvenBlockHeader> GetAsync(int blockHeight)
         /// <inheritdoc />
         public void AddToPendingBatch(ProvenBlockHeader provenBlockHeader, HashHeightPair newTip)
         {
+            this.logger.LogTrace("({0}:{1},{2}:{3})", nameof(provenBlockHeader), provenBlockHeader, nameof(newTip), newTip);
```

Comment:
`''` are missing. 

I think it's better to enable trace logs in fody config instead of adding manual entry\exit logs